src/content/structured/get-started/figma.mdx

```diff
@@ -45,13 +38,21 @@ The UI Kit is turned on by default for all projects within our Figma organisatio
 2. Make sure the Intelligence Community Design System library is turned on.
 
 <DoubleDoDontCaution>
-  <DoDontCaution
+<DoDontCaution
     imageSrc={figmaFig2}
     imageAlt="Image of the Libraries menu in Figma with the Intelligence Community Design System library turned on."
     state="none"
   />
 </DoubleDoDontCaution>
 
-## How to use the Figma UI Kit components
+## Not part of the Intelligence Community?
+
+We've published our [Figma UI Kit](https://www.figma.com/community/file/1283021660431259186) to the Figma Community, so that everyone can use our resources.
+
+You can make a copy of it in your local files and use the components in your designs. This is a static only version and you won't automatically receive component updates. Check Figma Community regularly for new updates and versions.
+
+    1. Navigate to the [Intelligence Community UI Kit on Figma Community](https://www.figma.com/community/file/1283021660431259186).
+    2. Duplicate the library into your drafts and connect your project files.
+    3. Use the [swap libraries](https://help.figma.com/hc/en-us/articles/360039150413-Swap-components-and-instances) function in Figma to quickly swap your components from one version to the next.
```

Comment:
For some reason, this is showing up in a code block in the build
<img width="829" alt="Screenshot 2023-09-12 at 09 21 05" src="https://github.com/mi6/ic-design-system/assets/113986285/c6efbfb6-b795-4f7c-9f0e-b96b9ff53c8e">